modules/failure-handler/src/main/java/org/apache/ignite/internal/failure/FailureManager.java

```diff
@@ -119,7 +137,7 @@ public FailureContext failureContext() {
      * Processes failure accordingly to configured {@link FailureHandler}.
      *
      * @param failureCtx Failure context.
-     * @return {@code True} If this very call led to Ignite node invalidation.
+     * @return {@code true} If this call leads to Ignite node invalidation and {@code false} otherwise.
```

Comment:
Do we need this javadoc at all? It seems to be no different from the javadoc on the interface method

Reply:
Ok. that makes sense, I will remove it.